pwa/src/templates/mappingDetailTemplate/MappingDetailsTemplate.tsx

```diff
@@ -50,7 +91,56 @@ export const MappingDetailTemplate: React.FC<MappingDetailsTemplateProps> = ({ m
             handleToggleDashboard={{ handleToggle: toggleFromDashboard, isActive: !!dashboardCard }}
           />
 
-          <MappingFormTemplate mapping={getMapping.data} />
+          <div className={styles.tabContainer}>
+            <TabContext value={currentTabs.mappingDetailTabs.toString()}>
+              <Tabs
+                value={currentTabs.mappingDetailTabs}
+                onChange={(_, newValue: number) => {
+                  setCurrentTabs({ ...currentTabs, mappingDetailTabs: newValue });
+                }}
+                variant="scrollable"
+              >
+                <Tab className={styles.tab} label={t("General")} value={0} />
+                <Tab className={styles.tab} label={t("Test Mappings")} value={1} />
+              </Tabs>
+
+              <TabPanel className={styles.tabPanel} value="0">
+                <MappingFormTemplate mapping={getMapping.data} />
+              </TabPanel>
+
+              <TabPanel className={styles.tabPanel} value="1">
+                <form onSubmit={handleSubmit(onSubmitTest)}>
+                  <Button
+                    className={clsx(styles.buttonIcon, styles.testConnectionButton)}
+                    disabled={isLoading.mappingForm}
+                    type="submit"
+                  >
+                    <FontAwesomeIcon icon={faArrowsRotate} />
+                    {t("Test simple mapping")}
+                  </Button>
+
+                  <div className={styles.content}>
+                    <FormField>
+                      <FormFieldInput>
+                        <FormFieldLabel>{t("Input")}</FormFieldLabel>
```

Comment:
I think some more information here would be very helpful. We could possibly use a info icon with a tooltip, like we do in the generated object forms.